src/store/modules/todo.module.js

```diff
@@ -0,0 +1,50 @@
+const state = {
+  items: [
+    { id: 1, name: "Jack", isEdit: false },
+    { id: 2, name: "Petter", isEdit: false },
+    { id: 3, name: "Tom", isEdit: false },
+  ],
+};
+const mutations = {
+  Add_Items: (state, payload) => {
+    state.items.push(payload);
+  },
+
+  TO_EDIT(state, payload) {
+    for (let i = 0; i < state.items.length; i++) {
+      if (i == payload) {
+        state.items[payload].isEdit = true;
+      } else {
+        state.items[i].isEdit = false;
+      }
+    }
+  },
+  UPDATE_newItems: (state, payload) => {
```

Comment:
naming case should be same, all in capital like TO_EDIT for mutation to distinguish and code quality.